src/Test/TestQueries.cs

```diff
@@ -0,0 +1,660 @@
+using System;
```

Comment:
This file has become too large, we need to split this file. We can do later also. Just adding comment for your reference. This becomes difficult to maintain and read.